pkgs/by-name/up/uppaal/package.nix

```
@@ -0,0 +1,88 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  makeWrapper,
+  jdk17,
+  makeDesktopItem,
+  copyDesktopItems,
+  wrapGAppsHook,
+  shared-mime-info,
+  gdk-pixbuf,
+}:
+
+stdenv.mkDerivation rec {
```

I suppose this might not need a C toolchain during buildtime?

```suggestion
stdenvNoCC.mkDerivation rec {
```